client/src/components/landing/index.js

```diff
@@ -24,6 +24,7 @@ const BigCallToAction = () => (
 
 export const Landing = ({ edges }) => {
   const superBlocks = uniq(edges.map(element => element.node.superBlock));
+  const interviewPrep = superBlocks.splice(6, 1);
```

Comment:
We should probably check that this one is going to come out to be Interview Prep. I think if we added a certification in future, this will differ?

A simple way to know would be adding a simple test and checking the DOM renders the Interview prep. superblock title.

In future any change would break the test drawing attention to the necessary change here.

Reply:
Yea, there should probably be something more there. Even if it's just a check in the code.

Reply:
I think adding some tests should be fine, so that this is not missed in future.